scripts/valhalla_build_extract

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+import argparse
+from io import BytesIO
+import json
+import logging
+from pathlib import Path
+import struct
+import sys
+import tarfile
+from tarfile import BLOCKSIZE
+from typing import Dict
+
+description = """Builds a tar extract from the tiles in mjolnir.tile_dir to the path specified in mjolnir.tile_extract."""
+
+# "=" prefix means native byte order with standard size and no alignment:
+# https://docs.python.org/3/library/struct.html#byte-order-size-and-alignment
+STRUCT_FORMAT = '=LQ'
```

Comment:
this confuses me a little, lacking the solid CS background I guess.. this is the byte layout specifier for struct to create a uint32_t and a uint64_t which results in 12 byte packets for two numbers: https://docs.python.org/3/library/struct.html#byte-order-size-and-alignment, the comment says it all. is this the way how to make sure any consumer architecture can read any other producer's architecture's index.bin file with this layout (e.g. I produce an extract on x86_64 and someone with ppc64 can read it)? or is there some more trickery involved in python or c++ to make sure endianess etc is properly taken care of?

Reply:
we should be using little endian for everything. little endian won the war, any platform we target will will be LE.

there are things we need to do in c++ to be robust to running on a different architecture than where the data was created but we've sided stepped that and pretty much assume little endian everywhere.

Reply:
I think it maybe it should be `<LQL`, which would have a matching c++ struct of:

```c++
struct tile_entry {
  uint32_t tile_id; //just level and tileindex hence fitting in 32bits
  uint64_t offset; //number of bytes offset from the beginning of the tar to the beginning of this tile
  uint64_t size;   //size of the tile in bytes
};
```